lib/common_test/src/ct_suite.erl

```diff
@@ -113,7 +112,7 @@ If `{skip, Reason}` is returned, all test cases in the module are skipped and
 For details on groups, see section
 [Test Case Groups](write_test_chapter.md#test_case_groups) in the User's Guide.
 """.
--doc(#{title => <<"Callback Functions">>}).
+-doc(#{}).
```

Comment:
```suggestion
```

Reply:
We may aswell delete all of these empty `-doc` attributes as they serve no purpose.